gradle.properties

```
@@ -17,9 +17,11 @@ POM_URL=https://github.com/requery/sqlite-android/
 POM_SCM_URL=https://github.com/requery/sqlite-android/
 POM_SCM_CONNECTION=scm:git:git://github.com/requery/sqlite-android.git
 POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/requery/sqlite-android.git
+POM_LICENCE_NAME=The Apache Software License, Version 2.0
```

https://spdx.org/licenses/Apache-2.0.html

The Full Name should be Apache License 2.0 or if you want to go with the shorter one, Apache-2.0

```suggestion
POM_LICENCE_NAME=Apache License 2.0
```

```suggestion
POM_LICENCE_NAME=Apache-2.0
```